web/src/components/troubleshoot/GenerateSupportBundle.jsx

```diff
@@ -23,12 +24,10 @@ class GenerateSupportBundle extends React.Component {
       totalBundles: null,
       showRunCommand: false,
       isGeneratingBundle: false,
-      initialLoading: true,
       displayRedactorModal: false,
       loadingSupportBundles: false,
       supportBundles: [],
       listSupportBundlesJob: new Repeater(),
-      pollForBundleAnalysisProgress: new Repeater(),
```

Comment:
we will not be polling for bundle analysis progress on this component anymore. This component will no longer show the progress bar. 
it will be done in `TroubleShootContainer.jsx` since `SupportBundleList` and `SupportBundleAnalysis` both need this polling function.